spec/models/place_bid_spec.rb

```diff
@@ -75,7 +80,91 @@
     end
   end
 
-  context 'when the bid amount is too low' do
+  context 'when the bid amount is above the current bid price' do
+    let(:auction) {
+      Presenter::Auction.new(Auction.create({
+        start_datetime: Time.now - 3.days,
+        end_datetime: Time.now + 7.days
+      }))
+    }
+    let(:amount) { 405 }
+
+    it 'should raise an authorization error' do
+      allow_any_instance_of(Presenter::Auction).to receive_message_chain(:current_bid, :amount).and_return(400)
```

Comment:
Needing a message stub chain seems like a smell. Wondering if current bid could just return a mock object. But there might be something simpler than this too. Sorry can't be of more help from github. If you want me to download and play around happy to. I am guessing that this is where folks on the Ruby channel were having concerns?